src/sage/combinat/cluster_algebra_quiver/cluster_seed.py

```diff
@@ -332,7 +332,7 @@ def __init__(self, data, frozen=None, is_principal=False, user_labels=None, user
                 else:
                     labelset = set(user_labels)
                     # Sanitizes our ``user_labels`` to use Integers instead of ints
-                    user_labels = [ZZ(x) if x in ZZ else x for x in user_labels]
+                    user_labels = [Integer(x) if x in ZZ else x for x in user_labels]
```

Comment:
What's the purpose of this change?

Reply:
this is marginally faster

Reply:
Then shouldn't we be using `Integer` throughout this PR instead of `ZZ`?